plugins/tinymcespellchecker.md

```diff
@@ -47,68 +49,186 @@ tinymce.init({
+
+**Default Value:** `false`
+
+**Possible Values:** `true`, `false`
+
+#### Example: `spellchecker_on_load`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_on_load: true
+});
+```
 
 ### `spellchecker_active`
-This option helps in deciding if the spellchecker should be initialized as active or not. With this set to `false` the spellchecker will not be activated automatically on text input. The toolbar button or the menu item will have to be pressed to start the spellchecking. Only applicable when using the default as-you-type spellchecking, not while using the dialog mode. Defaults to `true`.
+
+This option enables or disables the spell checker when the editor is loaded. When set to `false`, the spellchecker will not be activate when the editor is initialized. The toolbar button or the menu item will have to be selected by the user to start the spell checker.
+
+**Type:** `Boolean`
+
+**Default Value:** `true`
+
+**Possible Values:** `true`, `false`
+
+#### Example: `spellchecker_active`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_active: true
+});
+```
 
 ## Toolbar Buttons
 
-### `spellchecker`
+The following toolbar buttons are provided by the Spell Checker Pro plugin:
 
-This button allows the user to perform a spellcheck on the entire document. In addition, the drop-down menu attached to this button allows specifying the language that is currently used when spellchecking. For more information about customizing the toolbar, refer to the [Editor appearance section of the documentation]({{site.baseurl}}/configure/editor-appearance/#toolbar).
+| Control              | Description                                                                           |
+| -------------------- | ------------------------------------------------------------------------------------- |
+| language             | Sets the spelling checker language for the current selection.                         |
+| spellcheckdialog     | Opens the spelling checker dialog.                                                    |
+| spellchecker         | Spellchecks the current editor contents.                                              |
 
-Example {{site.productname}} Configuration:
+For more information about customizing the toolbar, see: [User interface options --- Toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar).
 
-````
+#### Example: Spell Checker Pro toolbar buttons
+
+````js
 tinymce.init({
   selector: 'textarea',
   plugins: 'tinymcespellchecker',
-  toolbar: 'spellchecker',
+  toolbar: 'spellchecker language spellcheckdialog',
   spellchecker_rpc_url: 'localhost/ephox-spelling',
-  spellchecker_language: 'en' runs in the background of all applications and spell checks everything that is typed in real-time, everywhere: online, off-line, in dialog boxes, in console windows, etc., to make the content mistake-free.
+  spellchecker_language: 'en'
```

Comment:
Should this be `en_us` or even just removed since it's the default?

Reply:
fixed